spec/src/main/asciidoc/license-efsl.adoc

```diff
@@ -25,8 +25,8 @@ document, or portions thereof, that you use:
 * link or URL to the original Eclipse Foundation document.
 * All existing copyright notices, or if one does not exist, a notice
   (hypertext is preferred, but a textual representation is permitted)
-  of the form: "Copyright (c) [$date-of-document]
-  Eclipse Foundation, Inc. \<<url to this license>>"
+  of the form: "Copyright (c) [{revdate}]
```

Comment:
I don't know if this substitution is required.  It's meant as an example.  By filling in the date and the specific url, then this might be used verbatim and it might not be accurate.  At least by leaving the "substitution text" users would know what is expected.  That's how I have been reading this section.